Back-end/schema/mutations.js

```diff
@@ -0,0 +1,38 @@
+const { UserDTC,DriverTC, OwnerTC, ParkingTC, RentTC, ConversationTC, MessageTC  } = require('./relations')
+// Getting objects after the relations are applied
+const mutations = {
+    // creates users
+    driverCreate: DriverTC.getResolver('createOne'),
+    ownerCreate: OwnerTC.getResolver('createOne'),
+    userCreate: UserDTC.getResolver('createOne'),
+
+    // creates
```

Comment:
"creates" what?

I think the most organized way to do this would be to use multiple objects and then merge them with spread syntax.

Example:
```
const UserMutations = {
  driverCreate: DriverTC.getResolver('createOne'),
  ownerCreate: OwnerTC.getResolver('createOne'),
  userCreate: UserDTC.getResolver('createOne'),
};

const ParkingMutations = {
  parkingCreate: ParkingTC.getResolver('createOne'),
  parkingCreateMany: ParkingTC.getResolver('createMany'),
  parkingUpdate: ParkingTC.getResolver('updateOne'),
};

module.exports = {
  ...UserMutations,
  ...ParkingMutations,
};
```

This way the code is self-documented and you don't need to add inline comments